tests/api/test_database_connection.py

```diff
@@ -13,8 +14,10 @@ def test_no_elastic(flask_api_url, docker_client):
     assert True
 
 
-def test_empty_database(flask_api_url):
+def test_empty_database(flask_api_url, mongo_client):
     """Sample test."""
+    result = mongo_client.projects.delete_many({})
+    print("Database cleaned: ", result)
     response = requests.get(flask_api_url + "/api/projects")
     assert response.text == "There are no projects"
```

Comment:
Although this test is correct, I am not certain this is the best way handle this case. I prefer the response to be an empty json array in that cases and the error code to be 200, could you quickly change that?